src/main/java/com/bookstore/entity/User.java

```diff
@@ -31,4 +40,44 @@ public class User {
     private String lastName;
 
     private String shippingAddress;
+
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "users_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles;
```

Comment:
```suggestion
    private Set<Role> roles = new HashSet<>();
```

Reply:
Incompatible types. Found: 'java.util.Set<com.bookstore.entity.Role>', required: 'java.util.Collection<? extends org.springframework.security.core.GrantedAuthority> 
I couldn't fix this error